plugin/src/main/java/com/exadel/aem/toolkit/plugin/utils/NamingUtil.java

```diff
@@ -120,14 +122,18 @@ public static String getUniqueName(String value, String defaultValue, Element co
     }
 
     /**
-     * Retrieves the member name value with the Java getter prefix stripped
-     * @param value String used as the source
+     * Retrieves the member name value with the Java getter prefix stripped for method, otherwise returns a full source
```

Comment:
(cosmetic) `Retrieves the member name that matches the given {@code Source}. If the source reflects a Java method, the {@code get...} prefix is stripped. Otherwise, the name is returned as is
@param source {@code Source} object reporting the name`